ush/set_extrn_mdl_params.sh

```diff
@@ -299,6 +306,12 @@ else
   esac
 
 fi
+
+
+if [[ "${DO_RETRO}" = "true" && ! -z "${EXTRN_MDL_SOURCE_BASEDIR_LBCS}" ]]; then
```

Comment:
Similarly, suggest to use double equal sign in this `if` statement

Reply:
Done

Reply:
Done